src/main/java/com/google/cloud/spanner/pgadapter/statements/IntermediatePreparedStatement.java

```diff
@@ -76,8 +80,18 @@ public void execute() {
     // don't need to do that once more.
     if (getStatementResult(0) == null) {
       try {
-        StatementResult result = connection.execute(this.statement);
-        this.updateResultCount(0, result);
+        // TODO: Refactor to use ClientSideStatement information.
```

Comment:
Could you also add buganizer Id's to the rest of the TODO's?

Reply:
Done